support/BRKey.c

```diff
@@ -364,7 +372,84 @@ size_t BRKeyLegacyAddr(BRKey *key, char *addr, size_t addrLen, BRAddressParams p
     return addrLen;
 }
 
-// signs md with key and writes signature to sig
+// convert signature from DER to raw encoding
+int BRSignatureDERToRaw (const uint8_t *derSig, size_t derSigLen, uint8_t *rawSig, size_t rawSigLen)
+{
+    const int sigLen = 64;
```

Comment:
I think we should avoid using signed types throughout here. I know the original Java used signed types, but unsigned types aren't readily available in Java. By sticking to unsigned types and verifying every bit of arithmetic for overflows, I think we will be better off.